source/style-guide/writing/use-active-voice.txt

```diff
@@ -4,16 +4,51 @@
 Use Active Voice
 ================
 
+.. default-domain:: mongodb
+
 .. include:: /includes/styles/corrections.rst
 
-Active voice makes the *performer* of the action (usually the user) the
-subject of the sentence. Passive voice makes the *recipient* of the
-action (not the performer) the subject of the sentence.
+.. _why-active-voice:
+
+Why Use Active Voice?
+---------------------
+
+- *Active voice* makes the *performer* of the action (usually the
+  reader or user) the subject of the sentence.
+
+- *Passive voice* makes the *recipient* of the action (not the
+  performer) the subject of the sentence.
+
+When compared to passive-voice sentences, active-voice sentences are:
+
+- more engaging
+- less complicated
+- less wordy
+- easier to understand
+
+With active voice, you can distinguish the actions and responses
+of the user from those of the technology.
 
-Active-voice sentences are more engaging, less complicated, less wordy,
-and easier to understand than passive-voice sentences. With active
-voice, the actions and responses of the user are easier to distinguish
-from the actions and responses of the technology.
+.. example::
+
+   Beginning users look for specific calls to action in procedures.
+   Active voice makes this clear.
+
+   Consider if you write a step that reads "The service can be
+   restarted".
+
+   As written, this sentence reads as a *consideration* or a
+   *possibility*, not an *instruction*. Let's said that, at this point,
+   if the user doesn't restart the service, the entire tutorial fails.
```

Comment:
I would be simpler about this:

"If the user skips this step due to this confusion, the rest of the tutorial is likely to fail."